### ISSUE TITLE
Styles tab issues

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#363]
1. [bug?] When I click the new Delete button to reset the Default config to standard, the Styles are not reset. Seems like this ought to reset the Styles tab settings to default, or else a Reset button on the Styles tab would be really great to have.

2. [bug] The superscript Raise value is not working when I set it in the Styles tab. It works in ptxprint-mods.sty.

3. [enhancement] The styles with levels (q q1 q2 etc) all get defined separately. I don't know if there's a way to carry my customizations down to the next level or farther for these leveled markers. For example, q and q1 should have identical formatting. I feel that this should be enforced in ptxprint, if possible. Even if I'm only using one level, q, in my project, this is likely to change as a translation project progresses and its markup is refined. And there might be both q and q1 markers in the text. It's not the responsibility of a typesetter-person to clean up the markers; this should be done before it gets to ptxprint, but since its purpose is to do speedy layout, it would be nice if this marker anomaly could be ignored. 

In addition, if I set smaller indent values for q1 it would be very helpful to carry those values down through the other levels. Then I only need to reduce one value, 1st line indent, on each level. Especially helpful as I can view details for only one style at a time, unlike when I manually edit a custom.sty file. I guess one way to override this would be to carry down the values only if I edit the unmarked style (e.g., \q). A popup indicating that I should check the other levels would also be helpful if this happens.

4. [question] Where are the styles zCopyRight, zLicense, zImageCopyrights defined? Are these user-editable? 

5. [minor annoyance] Could zCopyright **not** have an uppercase 'R', please? Copyright is a single word, and this tweak would make the name parallel to zImageCopyrights?

6. [comment/encouragement] Thank you for the tooltip about the usage of the various .sty files!! Very helpful!

**Comment by markpenny** [comment]
1. Yes, we discovered that bug yesterday and fixed it - this will fix many other similar problems in the Styles are. It will be available in 1.5.8.5 or 1.5.9
2. I need to look into this (it might be similar to 1. above, but could still be its own separate issue)
3. I like the idea - at least to connect/equate \q and \q1; \s and \s1 etc; but I don't know how hard that is to pull off. We'll have a think about it, but don't hold your breath! (I think the additional idea of "carrying those values down through the other levels" is much harder to do, and probably not practical as we can't read the user's mind about WHICH parameters should carry down and which ones shouldn't. And given that this is a pretty rare process (of setting up) even if we worked out how to do so, I'm not sure the cost would justify the gain.
4. **\\zCopyright** and **\\zLicense** are both read from the PT Settings file BUT **can be edited manually on the Basic tab**.
![image](https://user-images.githubusercontent.com/12962708/106326012-a8419a00-62e0-11eb-83b3-3e0e4b4b093b.png)
Then they are "pulled in" to the colophon using these codes on the Layout tab:
![image](https://user-images.githubusercontent.com/12962708/106326327-29992c80-62e1-11eb-92b4-e73888d295a2.png)

**\\zImageCopyrights** isn't editable because it is auto-generated from a file the XeTeX creates when typesetting (and hence the need to re-run Print in order to pick up those details and populate the rest of the colophon).
5. I'm not seeing where the upper case **R** is in \\zCopyright. Could it be that it is defined that way in Paratext? Look in Project Properties...
6. Glad that it is helpful. You can thank John N for the suggestion of documenting it somewhere; and I'm eager to keep all of the needed help at people's fingertips (Tooltips) rather than hiding it somewhere else online in a document that no one can find, read or keep updated. This strategy is now paying off as Localization gets underway and all the help/tooltip information is included in the materials to be translated.

**Comment by mhosken** [comment]
2. Superscript and raise don't work together. If you want to control the amount and size of your superscript, set the font size and raise and don't select superscript. Superscript is a lazy man's way to just get a default size and raise and is a bit of a legacy thing.

**Comment by KimB2017** [comment]
2. OK, I see this now. But maybe a tooltip about this would be helpful? Compared to the .sty method for making a \v smaller, when Superscript is turned off in Styles I have to set a much smaller size factor and a much larger Raise factor than when I specified the same "look" in ptxprint-mods.sty

**Comment by KimB2017** [comment]
4. Oh yeah, I forgot about the Base tab! This is good, but could the default "Wycliffe, Inc." have a © added, please, since this character is not easily typeable? It's easier to erase than to type a missing one.
